### PR TITLE
ISPN-14079 Attach sources and javadoc to Jakarta artifacts

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-14079

### CODE REVIEW

File: build/configuration/pom.xml [hunk 1]
```diff
@@ -307,11 +313,87 @@
...
    </build>
 
    <profiles>
+      <profile>
+         <id>jakarta-transform</id>
+         <activation>
+            <file>
+               <exists>${basedir}/jakarta.txt</exists>
+            </file>
+         </activation>
```
Comment (jabolina) [inline build/configuration/pom.xml:332]: really neat

File: cdi/common-jakarta/pom.xml [hunk 2]
```diff
@@ -14,6 +14,12 @@
     <artifactId>infinispan-cdi-common-jakarta</artifactId>
     <name>Infinispan CDI Jakarta EE support common</name>
 
+    <properties>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```
Comment (jabolina) [inline cdi/common-jakarta/pom.xml:18]: by skipping won't make ossrh to complain?

File: commons/all-jakarta/pom.xml [hunk 3]
```diff
@@ -76,34 +82,42 @@
...
-                        <jakartaDefaults>true</jakartaDefaults>
-                    </rules>
-                </configuration>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                        <phase>package</phase>
+    <!--profiles>
```
Comment (jabolina) [inline commons/all-jakarta/pom.xml:85]: Do we remove this?

File: cdi/common-jakarta/pom.xml (hunk 2 above)
Comment (tristantarrant) [reply cdi/common-jakarta/pom.xml:18]: No, because it is being attached by the transformer plugi

File: commons/all-jakarta/pom.xml (hunk 3 above)
Comment (tristantarrant) [reply commons/all-jakarta/pom.xml:85]: Right, let's wait for CI and then I'll remove that

**Comment by tristantarrant** [comment]
@jabolina , CI was good. So I removed the commented bit. You can merge

**Comment by jabolina** [comment]
Merged, thanks @tristantarrant

**Pull request merged**